application/logger.py

```
@@ -0,0 +1,128 @@
+            self.ostream.flush(FLUSH_FRAME)
+            self.ostream.write(EOF_CHAR)
+        finally:
+            self.ostream.close()
+
+        # Generate a new log filename
+        new_log_file = self._generate_log_filename()
+
+        # Initialize the new stream
+        try:
+            new_stream = open(new_log_file, "ab")
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)